### DISCUSSION TITLE
GitHub Reporting Workouts Happened 2 hours later to Garmin

### DISCUSSION THREAD

**Discussion by Buccafresh** [philosowaffle/peloton-to-garmin#561] [Q&A] [answered]
Let me start by saying I am having SO MUCH fun learning this new tool.

So far I have learned Windows Executable on my Windows machines, and I tried to figure out docker on my Mac (but I was too confused) so then I attempted to figure out the GitHub actions for my Mac.

One thing I noticed is that when I sync via the GitHub Actions, the workouts show that they were completed at two hours after it was completed. So if I completed a ride at 3:54, it will say that was completed at 5:54. 

I noticed using the Windows Executable / Peloton Console on Windows, this bug is nonexistent, the times it said an activity was completed are correct and true.

It's really a very small thing in the grand scheme of things, but I just wanted to point it out in case it was a super quick fix. Please let me know if setting the time is something I need to change in my GitHub profile somehow, or if this is a coding thing.   

I attached a photo to show you what I am referring to.
![time](https://github.com/philosowaffle/peloton-to-garmin/assets/150507039/b8616777-af9c-49c2-b195-07c7c79aabf5)

**Comment by philosowaffle** [comment, marked as answer]
[Is this](https://github.com/Buccafresh/peloton-to-garmin/blob/master/.github/workflows/sync_peloton_to_garmin.yml#L90) the correct Timezone for you?  If not, start by adjusting that field and see if it fixes it for you.

**Reply by Buccafresh** [reply]
Thank you, I was able to change it to Los Angeles for my Pacific (Portland, OR) time zone. That fixed it. Brilliant! Thank you.
